docs/pages/includes/metrics.mdx

```diff
@@ -1,59 +1,67 @@
-| `grpc_server_handled_total`                     | counter     | Teleport Auth       | Total number of RPCs completed on the server, regardless of success or failure.                    |
-| `grpc_server_msg_received_total`                | counter     | Teleport Auth       | Total number of RPC stream messages received on the server.                                        |
-| `grpc_server_msg_sent_total`                    | counter     | Teleport Auth       | Total number of gRPC stream messages sent by the server.                                           |
-| `heartbeat_connections_received_total`          | counter     | Teleport Auth       | Number of times the Auth Service received a heartbeat connection.                                  |
-| `s3_requests_total`                             | counter     | Amazon S3           | Total number of requests to the S3 API.                                                            |
-| `s3_requests`                                   | counter     | Amazon S3           | Total number of requests to the S3 API grouped by result.                                                        |
-| `s3_requests_seconds`                           | histogram   | Amazon S3           | Request latency for the S3 API.                                                                    |
-| `teleport_audit_emit_events`                    | counter     | Teleport Audit Log  | Number of audit events emitted.                                                                    |
-| `teleport_connected_resources`                  | gauge       | Teleport Auth       | Number and type of resources connected via keepalives.                                             |
-| `teleport_registered_servers`                   | gauge       | Teleport Auth       | The number of Teleport services that are connected to an Auth Service instance grouped by version. |
-| `user_login_total`                              | counter     | Teleport Auth       | Number of user logins.                                                                             |
-| `teleport_migrations`                           | gauge       | Teleport Auth       | Tracks for each migration if it is active (1) or not (0).                                          |
-| `watcher_event_sizes`                           | histogram   | cache               | Overall size of events emitted.                                                                    |
-| `watcher_events`                                | histogram   | cache               | Per resource size of events emitted.                                                               |
+| Name                                                     | Type      | Component           | Description                                                                                        |
```

Comment:
Is there a way to reduce the size of the diff in this partial? I'm not really sure what content this PR adds here.

Reply:
@ptgott you can use hide whitespaces changes in diff 
<img width="411" alt="Screenshot 2023-05-25 at 08 50 06" src="https://github.com/gravitational/teleport/assets/14020794/0b485f37-05f6-495f-9d6f-34655a4fe65c">

with that settings you can see that there are added only new metrics and formatting update due to size of first colum:
```
| `teleport_audit_parquetlog_batch_processing_seconds`     | histogram | Teleport Audit Log  | Duration of processing single batch of events in parquet audit log.                                |
| `teleport_audit_parquetlog_s3_flush_seconds`             | histogram | Teleport Audit Log  | Duration of flushing parquet files to s3 in parquet audit log.                                     |
| `teleport_audit_parquetlog_delete_events_seconds`        | histogram | Teleport Audit Log  | Duration of deletion events from SQS in parquet audit log.                                         |
| `teleport_audit_parquetlog_batch_size`                   | histogram | Teleport Audit Log  | Overall size of events in single batch in parquet audit log.                                       |
| `teleport_audit_parquetlog_batch_count`                  | counter   | Teleport Audit Log  | Total number of events in single batch in parquet audit log.                                       |
| `teleport_audit_parquetlog_last_processed_timestamp`     | gauge     | Teleport Audit Log  | Number of last processing time in parquet audit log.                                               |
| `teleport_audit_parquetlog_age_oldest_processed_message` | gauge     | Teleport Audit Log  | Number of age of oldest event in parquet audit log.                                                |
| `teleport_audit_parquetlog_errors_from_collect_count`    | counter   | Teleport Audit Log  | Number of collect failures in parquet audit log.    
```